watertap/examples/flowsheets/RO_with_energy_recovery/RO_with_energy_recovery.py

```diff
@@ -459,8 +500,8 @@ def optimize_set_up(m):
     # RO
     m.fs.RO.area.unfix()
     m.fs.RO.area.setlb(1)
-    m.fs.RO.area.setub(150)
-
+    m.fs.RO.area.setub(1500)
+    m.fs.RO.feed_side.friction_factor_darcy[...].setub(None)
```

Comment:
It seems necessary for some of the sweep analysis to get a solution, I am curious what is the original upper bound and what if we keep it, does most sweep runs fail?

Reply:
The original upper bound appears to be 5, and you're right that if we remove this upper bound, the flowsheet is still able to solve, but the sweeps will fail (specifically those at 90% recovery)